vendor/desktop-notifications/README.md

```diff
@@ -0,0 +1,98 @@
+# desktop-notifications
+
+## A simple and opinionated library for working with OS notifications
+
+## Goals
+
+- zero dependencies
+- good support for Windows notifications
+- leverage TypeScript declarations wherever possible
+
+**Note:** This is currently in preview, with support for features that GitHub
+Desktop uses.
+
+## Install
+
+```shellsession
+$ npm install --save desktop-notifications
+# or
+$ yarn add desktop-notifications
+```
```

Comment:
We can get rid of this right?

Reply:
Removed in [e0d8cb6](https://github.com/desktop/desktop/pull/20248/commits/e0d8cb6d55d1ed1e349682b78c1f308e326aacc0)